modules/kubernetes/pages/k8s-operator/backup-and-restore.adoc

```diff
@@ -0,0 +1,769 @@
+
+For example, to change the schedule to back up once per day at 12:00:
+
+[source.wrap, console]
+----
+kubectl tg backup-schedule update --cluster-name test-cluster -n tigergraph \
+--tag localdaily --schedule '0 12 * * *'
+----
+
+[NOTE]
+If there is a backup job running when you change the configuration, the running job won’t be affected.
+The new configuration will take effect in the next schedule.
+
+=== List all backup schedules
+You can list all existing backup schedules in a namespace:
+
+[source.wrap, console]
+----
+kubectl tg backup-schedule list --namespace tigergraph
+----
+
+==== Delete a backup schedule
+
+[source.wrap, console]
+----
+kubectl tg backup-schedule delete --cluster-name test-cluster --tag daily \
+--namespace tigergraph
+----
+
+=== Show backup schedule status
+
+[source.wrap, console]
+----
+kubectl tg backup-schedule status --cluster-name test-cluster --tag daily --namespace tigergraph
+
+
+Name:         test-cluster-schedule-daily
+Namespace:    default
+Labels:       <none>
+Annotations:  <none>
+API Version:  graphdb.tigergraph.com/v1alpha1
+Kind:         TigerGraphBackupSchedule
+Metadata:
+Creation Timestamp:  2022-12-20T02:40:10Z
+Generation:          1
+Resource Version:  1696649
+UID:               f8c95418-bcb3-495b-b5e4-5083789ce11a
+Spec:
+Backup Template:
+Backup Config:
+Compress Process Number:  0
+Tag:                      daily
+Timeout:                  18000
+Cluster Name:               test-cluster
+Destination:
+Local:
+Path:   /home/tigergraph/backup
+Storage:  local
+Schedule:     * * * * *
+Status:
+Conditions:
+Last Transition Time:  2022-12-20T02:42:01Z
+Message:               Backup job is active
+Reason:                BackupActive
+Status:                True
+Type:                  Active
+Job Counter:
+Successful Jobs:     1
+Last Schedule Time:    2022-12-20T02:42:00Z
+Last Successful Time:  2022-12-20T02:41:11Z
+
+Events:
+Type    Reason                   Age                From                      Message
+----    ------                   ----               ----                      -------
+Normal  Backup schedule created  2m1s               TigerGraphBackupSchedule  Create a new backup schedule success.
+Normal  Backup job succeed       60s                TigerGraphBackupSchedule  Last scheduled job succeed
+Normal  Backup job created       10s (x2 over 71s)  TigerGraphBackupSchedule  Schedule a new backup job
+----
+
+The Events section tells you whether the scheduled job was successful.
+
+== Pause or resume a backup schedule
+
+To pause a running backup schedule:
+
+[source.wrap, console]
+----
+kubectl tg backup-schedule pause --cluster-name test-cluster -n tigergraph \
+--tag localdaily
+----
+
+The next backup job will not be scheduled until the schedule is resumed.
+
+To resume a paused schedule:
+
+[source.wrap, console]
+----
+kubectl tg backup-schedule resume --cluster-name test-cluster -n tigergraph \
+--tag localdaily
+----
+
+
+== Backup controls
```

Comment:
```suggestion
== Backup restrictions
```

Reply:
"Control" is a very general therefore not very descriptive word.  Since all of these options set a maximum, perhaps "restrictions" is better. "limits" is another option. I'm open to alternatives; I'm just not satisfied with "control".